umm-lib/src/cmr/umm/echo10/related_url.clj

```diff
@@ -9,7 +9,8 @@
 
 (def resource-type->related-url-types
   "A mapping of ECHO10 OnlineResource's type to UMM RelatedURL's type and sub-type.
-  This came from a list provided by Katie on ECHO10 collections, more may need to be added for granules."
+   This came from a list provided by Katie on ECHO10 collections, more may need to
```

Comment:
The convention is actually lining up with the double quote above. I used to do what you're doing and was asked to change.

Reply:
I thought about this and thought I got it correct. I'll fix it.